packages/base-contract/src/globals.d.ts

```diff
@@ -4,3 +4,5 @@ declare module '*.json' {
     export default json;
     /* tslint:enable */
 }
+
+declare module 'ethereumjs-vm';
```

Comment:
Can we move this to `typescript-typings` package and write the types for the methods we use of this dep?